.github/workflows/ci.yml

```diff
@@ -25,11 +25,3 @@ jobs:
           CI: 'true'
       - name: Test
         run: yarn test
-      # To do: macos isn’t used anymore
-      - name: Publish CI tag to npm
-        if: matrix.node == 12 && matrix.os == 'macOS-latest' && github.event_name == 'push' && github.ref	== 'refs/heads/master'
-        run: |
-          npm set //registry.npmjs.org/:_authToken $NPM_AUTH_TOKEN
-          yarn publish-ci
-        env:
-          NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
Why remove the code for prepublishing, instead of fixing it?

Reply:
It is in `publish.yml` now.

Reply:
I was assuming that `publish.yml` was publishing tags, and that this file here was publishing canary releases?
Is there still something doing `publish` or `publish-next`?

Reply:
Not for now, I don't think you want to publish on CI automatically.

It can be changed to a manual triggered task if you want.

https://github.com/alauda/alauda-ui/blob/master/.github/workflows/publish.yml#L4

Take this for example.

Reply:
ok, sounds good!